mosec/coordinator.py

```diff
@@ -263,6 +284,8 @@ def coordinate(self):
         while not self.shutdown.is_set():
             try:
                 _, ids, payloads = protocol_recv()
```

Comment:
Since both normal server and sse server will fetch request from this `protocol_recv`, chances are that sse server will get data from `/inference` request when clients query the wrong endpoint. In this case, the server will process the data but the client will get nothing.

Similarly, if the client query a normal server with `/sse_inference`, weird thing will happen.

We'd better to handle this by sending error for now; for future we should not have both endpoints exposed by default, but use multi_route kind of feature to support only when the server implements it.

Reply:
1. impl sse but called /inference => cannot find the stream event channel (info log), but return nothing to the client

2. impl inference but called /sse_inference => processed but return nothing

Shall we return an error if it cannot find the stream event channel? This might not be 100% accurate.

Not sure how to deal with the 2. under the current design.

Reply:
Is it possible to do a check at server side, knowing whether it is a normal sever or sse server, then send this info to rust side. In doing so, the rust side will know whether it should reject the requests from non-implemented route.

A simple possible implementation is to let users set a class attribute to the sse worker? Only when it is set the `send_stream_event` can be called normally.

Reply:
This might need to implement an SSE worker class. But passing the type of worker might require a more complex config file than the current command line arguments.